class.jetpack.php

```diff
@@ -3930,10 +3930,11 @@ function admin_page_load() {
 			Jetpack::restate();
 		}
 
+		$from = ! empty( $_GET['from'] ) ? $_GET['from'] : 'iframe';
```

Comment:
But I also must say I agree with @kraftb in terms of the fact that I've always wanted us to have a whitelist of those against which to validate the `from` argument. Oh well 🤷‍♂️

Reply:
Should I add a `@todo` for this?

Reply:
Would be awesome @justinshreve, thanks!